### PR TITLE
Problem Framing in Machine Learning

### PR DESCRIPTION
**Pull request by collins-a**
Article as index.md
Hero image as hero.jpg

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #1183

File: articles/problem-framing/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Beginning with a simple model
+
+To allow for a not only explainable but interpretable model, one may consider using a simple model at this stage. A simple model may be described as one with the least possible complexity required to correctly carry out a task. The use of such a model allows for controllable debugging. Simple models also provide a good baseline. They can help evaluate whether there is a need for a more complex model. Complex models provide less explainability. They are harder and take longer to train compared to simple ones. If the need for more complexity is justified, it may be implemented later on.
+
+### Wrapping up
+
+Problem framing is a key step in the machine learning workflow. Since it’s arguably less technical than other stages in a machine learning workflow, it might be inaccurately perceived as an easy step. Yet, the challenges we covered show otherwise. The best practices we have covered should be of help when framing machine learning problems.
```
Comment (lalith1403) [inline articles/problem-framing/index.md:90]: You mention problem framing and then data analysis. So does data collection and understanding sources of data come under problem framing as well? Can you elaborate on some of the issues under data collection?

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written. I feel more information is required under a few sections. The content is a bit generic and can benefit from being more targeted and unique. Add more information wherever you feel appropriate to improve the article.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written. I feel more information is required under a few sections. The content is a bit generic and can benefit from being more targeted and unique. Add more information wherever you feel appropriate to improve the article.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by collins-a** [comment]
@lalith1403 thank you, as usual. I struggled to get in-depth with this topic because a great number of the sources I found were barebones. 

Still, I have added some content to various sections of the best practices. I hope it suffices. Cheers!

**Review by lalith1403 — APPROVED** [review]

**Pull request merged**